src/pages/Account/Account.scss

```diff
@@ -124,6 +134,16 @@ $page-max-width: 1240px;
     height: 225px;
     background-size: 280px;
   }
+
+  &.github {
+    margin: 0 auto;
+    background-image: url(/assets/images/Summit_Logo.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 580px;
+    height: 225px;
+    background-size: 280px;
```

Comment:
[NB] use scss variables

Reply:
the .hero-logo and .sparkle styles don't use variables, so i thought we shouldn't